app/collections/LevelSessions.js

```diff
@@ -58,6 +58,7 @@ module.exports = (LevelSessionCollection = (function () {
       let skip = 0
       const size = _.size(classroom.get('members'))
       if (options.data == null) { options.data = {} }
+      options.data.project = options.data.project || 'state.complete,state.introContentSessionComplete,state.goalStates,level,creator,changed,created,dateFirstCompleted,submitted,codeConcepts,code,codeLanguage,introContentSessionComplete,playtime'
```

Comment:
where did we find this projection? Sure, it will not break any existing stuff?

Reply:
from ozaria dashboard. and yes not break anything , I already deployed in china